src/distributions/uniform.rs

```diff
@@ -967,6 +974,26 @@ mod tests {
     use super::*;
     use crate::rngs::mock::StepRng;
 
+    //FIXME: instantiate a UniformDuration
+    // #[test]
+    // #[cfg(feature = "serde1")]
+    // fn test_serialization_uniform_duration() {
+    //     let distr = UniformDuration::new(-25, 50);
+    //     let de_distr: UniformDuration = bincode::deserialize(&bincode::serialize(&distr).unwrap()).unwrap();
+    //     assert_eq!(
+    //         distr.offset, de_distr.offset
+    //     );
+    //     //FIXME: check if distr.mode are the same
+    // }
+    
+    // #[test]
+    // #[cfg(feature = "serde1")]
+    // fn test_uniform_serialization() {
+    //     let unit_box = Uniform::new(-1, 1);
+    //     let de_unit_box: Uniform<i32> = bincode::deserialize(&bincode::serialize(&unit_box).unwrap()).unwrap();
+    //     //FIXME: assertion needed to check
+    // }
```

Comment:
Please check with both integer and float inputs.

One should be able to directly access internals here (e.g. `unit_box.0.low`).

Reply:
Bear in mind how `Uniform` works: the inner type is auto-detected, so e.g. `Uniform::new(-1, 1)` and `Uniform::new(-1.0, 1.0)` have distinct inner types.